### PR TITLE
Improved readme

### PR DESCRIPTION
**Pull request by ryanbnl**
Added description of files, governance and reminder over the gateway. Also renamed files for clarity based on other discussions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,35 @@
-# EU eHealthNetwork Digitial COVID Certificate value sets
+# EU eHealthNetwork Digital COVID Certificate value-sets
```
Comment (dslmeinte) [inline README.md:1]: "value-sets" or "value sets" (or "valuesets")? Any of these are fine, as long as it's consistent across the entire repo.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+All of the file found in this repository should be considered for reference usage only. The current official release is published on the EU DGC Gateway. As access to this gateway is only available to Member States please refer to your own Member State to for access to those files.
 
 Release: 2.11.0
 
-Note that these value sets do not form a core part of the [JSON Schema](https://github.com/ehn-dcc-development/ehn-dcc-schema) but are referenced by it. These value sets can (and should) be regularly updated and distributed from the main Digital COVID Certificate Gateway (DCCG), under clear versioning and release management.
+## Overview
+
+Here is an overview of the files available in this repository. The column "managed on" describes the place where the value-set is managed. In most cases that is in this Github repository. Those value-sets will usually be very close to the versions provided on the EU DGC Gateway.
+
+**Reminder: the value-sets on the EU DGC Gateway are leading, the value-sets here should only be used for development purposes.**
```
Comment (dslmeinte) [inline README.md:13]: Apart from the RAT stuff, this repo is (effectively) leading through process. Maybe we can reflect that?

Also: "EU D*C*C Gatewau", right?

**Review by dslmeinte — CHANGES_REQUESTED** [review]

**Comment by dslmeinte** [comment]
We should merge this PR into the `release/2.12.0` branch, not directly `main`, I think.

File: README.md (hunk 1 above)
Comment (ryanbnl) [reply README.md:1]: I changed it al to value-sets, but am happy with valuesets.

Comment (dslmeinte) [reply README.md:1]: Then keep it at "value-sets"!

File: README.md (hunk 2 above)
Comment (ryanbnl) [reply README.md:13]: It's called the EU DGC Gateway, https://github.com/eu-digital-green-certificates/dgc-gateway. I'm keeping it consistent with that, as when you google "EU DGC Gateway" you get the gateway and "EU DCC Gateway" gives the DCC docs.

Comment (dslmeinte) [reply README.md:13]: The 1st paragraph speaks about "EU DCC Gateway": let's change that for consistency. Agree with sticking to the leading sources.

**Review by dslmeinte — APPROVED** [review]

**Review by dslmeinte — APPROVED** [review]

**Review by skounis — APPROVED** [review]

**Review by skounis — COMMENTED** [review]
The name of the gateway is yet DCCG, so I would suggest using EU DCC Gateway

**Pull request merged**